python_frank_energie/models.py

```
@@ -5223,10 +4971,10 @@ def from_dict(cls, data: dict[str, object]) -> SmartBattery:
 
         capacity_value = data.get("capacity")
 
-        capacity: float | None = None
+        _capacity: float | None = None
         if capacity_value is not None:
             try:
-                capacity = float(capacity_value)
+                _capacity = float(capacity_value)
```

Flake8 (7.3.0)

[error] local variable '_capacity' is assigned to but never used

(F841)